### ISSUE TITLE
XML parsing problem of package.xml while installing via pear

### ISSUE THREAD

**Issue by DonSonic** [predis/predis#531]
I had to convert package.xml to UTF-8 to be able to install via PEAR on my Gentoo.

**Comment by tillkruss** [comment]
@DonSonic: Can you submit a PR to fix this?

**Comment by DonSonic** [comment]
I'm out, I don't use the software anymore and cannot provide any further details. As I said, the package.xml just has to be converted to UTF-8.

**Comment by nrk** [comment]
Sounds strange to me as [DOMDocument::saveXML()](https://github.com/nrk/predis/blob/v1.1.1/bin/create-pear#L205) should be outputting UTF-8 if not instructed otherwise. I'll have a look anyway, thanks @DonSonic for the hint on how you got that fixed.

**Comment by nrk** [comment]
Should be fixed now. Tested on the PEAR packages I just published for the recently released patch releases (`1.1.2`, `1.1.3`) and it seems to be working fine (provided that I still don't have any issue with old packages):

```
pear install nrk/predis-1.1.2
downloading Predis-1.1.2.tgz ...
Starting to download Predis-1.1.2.tgz (235,785 bytes)
.....................................done: 235,785 bytes
install ok: channel://pear.nrk.io/Predis-1.1.2
```

```
pear install nrk/predis-1.1.3
downloading Predis-1.1.3.tgz ...
Starting to download Predis-1.1.3.tgz (235,808 bytes)
.................................................done: 235,808 bytes
install ok: channel://pear.nrk.io/Predis-1.1.3
```

Thank you for reporting the issue.
